core/views.py

```diff
@@ -88,4 +89,17 @@ def four_random_cards(redis_con, network):
         tmp = ast.literal_eval(redis_con.srandmember(network + '_users'))
         if tmp not in users:
             users.append(tmp)
-    return users
\ No newline at end of file
+    return users
+
+def suggestion(request):
+    if request.method == 'POST':
+        form = SuggestionForm(request.POST)
+
+        if form.is_valid():
+            form.save(commit=True)
+            return HttpResponseRedirect('/login/')
+
```

Comment:
Since we are now going the route of the suggestion box in a modal. lets not have it redirect them anywhere after they submit. We'll just close the box